cypress/integration/seededFlows/podcastFlows/togglePodcastPlayback.spec.js

```diff
@@ -0,0 +1,18 @@
+describe('Toggle Podcast playback', () => {
+  beforeEach(() => {
+    cy.testSetup();
+  });
+
+  it('should toggle podcast playback', () => {
+    cy.visit('/pod');
+    cy.contains('div', 'Example media | crow call').click();
+
+    cy.get('#record-crow-call').as('toggleButton');
```

Comment:
```suggestion
    cy.findByRole('button', { name: 'Play podcast' }).as('toggleButton');
```

This test is looking great - thanks for adding! I small change that would help make this more robust would be to use the `findByRole` query to get the button by its accessible name - this helps make sure we test as close to genuine user interactions as possible 🙂

Reply:
I'm going ahead and merging this, so we can get ahead and check if the build still passes (which totally should)

Reply:
We would need to have this as `cy.findByRole('button', { name: 'Developer on Fire Play podcast' })` since we have two images inside the `button` element with "Developer on Fire" and "Play podcast" alt texts.

![Screenshot](https://user-images.githubusercontent.com/47032027/130219186-153385b3-3d70-4185-9462-50601b493520.png)

I have tested this, should I go ahead and update it with same?

Reply:
I spotted my mistake too and pushed the change up (thought I should clean up my own mess haha!). We should be all good now 🙏  Sorry about that!